src/commands/general/vote.ts

```diff
@@ -18,9 +17,7 @@ import { VoteEmbed } from '../../lib/embeds';
 })
 export class VoteCommand extends Command {
 	public override registerApplicationCommands(registry: Command.Registry) {
-		registry.registerChatInputCommand(VoteCMD(), {

```

Comment:
why removing the whole thing? undefined should be enough no? That way we can directly see, what the command is (in this case `global`)

Reply:
If this is not specified, it is necessarily global, we should only use it when it is not global

Reply:
i personally like this not that much but it will do for now i guess